README.md

```diff
@@ -200,20 +202,96 @@ Retrieve a fully generated sitemap for your blog.
 
 ## Content Fields
 
-As demonstrated in the [Content Fields documentation](https://buttercms.com/docs/content/), any number of user-defined content fields can be retrieved from the API. Expanding on the examples in the docs, https://api.buttercms.com/v2/content/?keys=homepage_headline,team_members&auth_token=321478403e868f0fc41f0115731f330ff720ce0b returns an object that won't fit neatly in one C# object. For this reason, the client forwards the JSON string response and leaves deserialization up to the caller.
+**New in version 1.3.0**
+
+By the power of .NET generics, Content Fields can now be deserialized by the libary! The former method that would defer deserialization is still available to ease transition.
 
 #### RetrieveContentFields() Parameters:
 | Parameter|Description|
 | ---|---|
 |keys|String array of desired keys|
+|parameterDictionary(optional)|Dictionary of additional parameters, such as "locale" or "test"|
+
+#### RetrieveContentFields() Exceptions:
+| Exception|Description|
+| ---|---|
+|[ContentFieldObjectMismatchException](#contentfieldobjectmismatchexception)|This exception will be thrown when the library can't fit the returned data into the passed object class. |
 
 #### Examples:
 ```C#
 var keys = new string[2] { "team_members[name=Elon]", "homepage_headline" };
-var contentFields = await butterClient.RetrieveContentFieldsJSONAsync(keys);
+var dict = new Dictionary&lt;string, string&gt;()
+            {
+                { "locale", "de" },
+                { "test", "1" }
+            };
+var teamMembersAndHeadline = butterClient.RetrieveContentFields<TeamMembersHeadline>(keys, dict);
```

Comment:
Can you explain how `TeamMembersHeadline` works? Is this the Generics `feature`?  Does it mean people need to define the class `public class TeamMembersHeadline` in their code to reflect the expected response and then the API response automatically gets deserialized into that object?

If so, could you add an example of defining `public class TeamMembersHeadline` to this README code sample?